SR2023/2023-01-26-southampton-march-tech-day.md

```diff
@@ -0,0 +1,29 @@
+---
+to: Student Robotics 2023 teams
+subject: Southampton March Tech Day
+---
+
+Hello,
+
+We're pleased to announce that we'll be hosting an [in-person Tech Day][event]
+on March 11th at the University of Southampton. Full details can be found on the
+[event page][event].
+
+Tech Days are opportunities for teams to spend a whole day working on their
+robot with lots of help available. They’re also an opportunity to see how other
+teams are doing or get more direct help with your robots.
+
+The aim of this day is to help you out by having guaranteed presence from us
+blueshirts. We'll be around to answer questions about the competition, give kit
+support, and assist with your strategy ideas.
+
+We see a strong correlation between teams to attend Tech Days and performance in
+the competition.
+
+Please signup by Friday 3rd March 2023, though we may close signups sooner if
+all places are filled. We will be confirming places shortly after you sign up.
```

Comment:
are we planning on emailing teams immediately after they sign up? Because that's what this last sentence is implying

Reply:
That's what we have been doing thus far (latency in the order of a day or so).